scripts/spreadsheet.py

```
@@ -0,0 +1,753 @@
+
+    parser.add_argument("-o", dest='out', action='store',
+                        default='stats.xlsx',
+                        help='save excel to file')
+    parser.add_argument("--threshold", dest='thresholdPercent', type=float,
+                        help="difference from average smaller than this percentage is ignored (not colorized).  Default %0.2f" % (thresholdPercent))
+    parser.add_argument("--low", dest='devMinThreshold', type=float,
+                        help="difference from average larger than this * stddev colored yellow.  Default: %0.2f" %(devMinThreshold))
+    parser.add_argument("--high", dest='devMinThreshold', type=float,
+                        help="difference from average larger than this * stddev colored red.  Default: %0.2f" %(devMaxThreshold))
+    parser.add_argument('-v', '--verbose', dest='verbose', default=0,
+                        action='count', help='verbosity of report: more data');
```

Ruff (0.14.4)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)